README.md

```diff
@@ -1,12 +1,40 @@
 # Missed Issues
 
-A CLI tool for finding tickets that mention a team by no one has responded to from the team.
+A CLI tool for finding issues that mention a team and no one from the team has responded to the issue.
 
-`missed-issues --org mapbox --team teamname`
+`missed-issues --org mapbox --team teamname --token github_token`
 
 This will list all issues in the mapbox org that mention `@mapbox/teamname` where no member of that team has replied to the issue since the mention.
 
 ## Flags
 
-- `--max-issues`: the max number of issues to return in the search result before filtering. Defaults to 100.
+**Required**
 
+- `--org`: the name of the org you want to search.
+- `--team`: the name of the team you are checking into. This MUST not contain the name of the org.
+- `--token`: a GitHub token with `read:org, repo` access
+
+**Optional**
+
+- `--from`: the start date of your search range. All issues returned will have been edited on or after this date. Defaults to one week ago.
+- `--ignore-repos` a comma-separated list of repos that you wish to exclude from your search results. These are excluding via filtering and will thus still affect your max-issues number.
+- `--max-issues`: the max number of issues to get before filtering to help avoid long requests. Defaults to 100. That said, hitting this max is bad for your results.
+- `--nonmembers`: a comma-separated list of user logins who should not be used to count a ticket as replied too.
+
+## Environment Variables
```

Comment:
Before this section, add a couple examples showing inputs and expected outputs.

Reply:
I'm going to do this right have the basic example at the start of the doc to make it up front and great :). No flags currently effect what the output looks like, just what is in the output.